bin/bank/pycbc_brute_bank

```diff
@@ -75,6 +76,7 @@ parser.add_argument('--tau0-crawl', type=float)
 parser.add_argument('--tau0-start', type=float)
 parser.add_argument('--tau0-end', type=float)
 parser.add_argument('--tau0-cutoff-frequency', type=float, default=15.0)
+parser.add_argument('--num_cores', type=int, default=1, help='number of cores for waveform generation parallelization.')
```

Comment:
@yi-fan-wang Can you match the option names to our other executables like pycbc_inference?

Reply:
change to use 'nprocesses' as in pycbc_inference